{
    "repository": "tbotnz/netpalm",
    "stars": 400,
    "language": "python",
    "size": 11223,
    "clone_url": "https://github.com/tbotnz/netpalm.git",
    "timestamp": "2023-06-28T15:02:07.642878Z",
    "clone_success": true,
    "number_of_actions": 1,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_build_tools": [
        "pytest"
    ],
    "actions_test_build_tools": [
        "pytest"
    ],
    "actions_run": {
        "failed": true,
        "tests": [],
        "stdout": "[096298ab-5132-48aa-bd54-319374511442/build] \ud83d\ude80  Start image=crawlergpt:latest\n[096298ab-5132-48aa-bd54-319374511442/build]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[096298ab-5132-48aa-bd54-319374511442/build]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[096298ab-5132-48aa-bd54-319374511442/build]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[096298ab-5132-48aa-bd54-319374511442/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1013 /tmp/ad9c0f7e-1596-11ee-8a50-bb14de238602/tbotnz-netpalm] user=0 workdir=\n[096298ab-5132-48aa-bd54-319374511442/build]   \u2601  git clone 'https://github.com/satackey/action-docker-layer-caching' # ref=v0.0.11\n[096298ab-5132-48aa-bd54-319374511442/build] \u2b50 Run Main actions/checkout@v2\n[096298ab-5132-48aa-bd54-319374511442/build]   \u2705  Success - Main actions/checkout@v2\n[096298ab-5132-48aa-bd54-319374511442/build] \u2b50 Run Main docker pull python:3.8-slim\n[096298ab-5132-48aa-bd54-319374511442/build]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/1] user= workdir=\n[096298ab-5132-48aa-bd54-319374511442/build]   | Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post \"http://%2Fvar%2Frun%2Fdocker.sock/v1.24/images/create?fromImage=python&tag=3.8-slim\": dial unix /var/run/docker.sock: connect: permission denied\n[096298ab-5132-48aa-bd54-319374511442/build]   \u274c  Failure - Main docker pull python:3.8-slim\n[096298ab-5132-48aa-bd54-319374511442/build] exitcode '1': failure\n[096298ab-5132-48aa-bd54-319374511442/build] \ud83c\udfc1  Job failed\n",
        "stderr": "Error: Job 'build' failed\n",
        "workflow": "/tmp/ad9c0f7e-1596-11ee-8a50-bb14de238602/tbotnz-netpalm/.github/workflows/testrun-crawler.yml",
        "build_tool": "pytest",
        "elapsed_time": 249.00180888175964
    }
}